members_2023.html

```diff
@@ -0,0 +1,765 @@
+
+<body>
+  <div class="loader-container">
+    <div class="spinner"></div>
+  </div>
+  <div class="upnav">
+    <nav role="navigation" class="navbar fixed-top navbar-expand-lg navbar-dark" style="background-color: black;">
+        <div data-aos="zoom-in-left" data-aos-duration="800" class="container-fluid ">
+            <a class="navbar-brand " href="./index.html"> <img
+                    src="WiOS_White_NoBG.png" class="mb-1 logo_img" height="30px" width="30px" alt="">&nbsp;WiOS</a>
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                    aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+
+                <ul data-aos="zoom-in-left" data-aos-duration="800" class="navbar-nav ms-auto mb-2 mb-lg-0">
+
+                    <li class="nav_item_mx-2">
+                        <a class="nav-link" aria-current="page"
+                            href="./index.html">Home</a>
+                    </li>
+                    <li class="nav_item_mx-2">
+                        <a class="nav-link" href="./index.html#about">About</a>
+                    </li>
+                    
+                    <li class="nav_item_mx-2">
+                        <a class="nav-link" href="./dist/index.html">Events</a>
+                    </li>
+                    <li class="nav_item_mx-2">
+                        <a class="nav-link" href="./LTS/index.html">LTS</a>
+                    </li>
+                    <li class="nav_item_mx-2">
+                        <a class="nav-link"
+                            href="./index.html#community">Community</a>
+
+                    </li>
+
+
+                    <li class="nav-item dropdown mx-2">
+                        <a class="nav-link dropdown-toggle">
+                            Members
+                        </a>
+                        <ul class="dropdown-menu">
+                            <li class="nav_item_mx-2"><a class="dropdown-item"
+                                href=#>2023 &nbsp; →</a>
+                            </li>
+                            <li class="nav_item_mx-2"><a class="dropdown-item"
+                                    href="./members_2022.html">2022 &nbsp; →</a>
+                            </li>
+                            <li class="nav_item_mx-2"><a class="dropdown-item"
+                                    href="./members_2021.html">2021 &nbsp; →</a>
+                            </li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</div>
+  <div class="lower_members mt-5 pt-4">
+    <h1 class="text-center text-light" data-aos="zoom-in-down">Members of 2021</h1>
```

Comment:
Please change this to `Members of 2023`

![image](https://github.com/WiOSc/Wios-New-Web/assets/107696874/c1860238-4bf5-45c6-8acd-097cabec2c65)